appengine/genetics/style.css

```diff
@@ -15,29 +15,343 @@
   position: relative;
 }
 
+#displayContent {
+  position: absolute;
+  width: 400px;
+  height: 400px;
+  overflow: hidden;
+}
+
 #display {
-  border: 1px solid #ddd;
-  vertical-align: bottom;
+  position: absolute;
+  width: inherit;
+  height: inherit;
+  background: #ccc;
+  z-index: -1;
+}
+
+.endGameDisplay {
+  position: absolute;
+  z-index: 1;
+  top: 25px;
+  left: 25px;
+  width: 350px;
+  height: 350px;
+  border-radius: 20px;
+  background: rgb(255, 255, 255);
+  background: rgba(255, 255, 255, 0.8);
 }
 
-#geneticsStatTable {
+.playerStatTable {
   width: 406px;
   max-width: 406px;
   margin-left: -2px;
   height: 20px;
 }
 
-#geneticsStatTable>tr>td {
+.playerStatTable>tbody>tr>td {
   padding: 0;
   border: 1px solid;
   background: #ccc;
   border-radius: 2px;
 }
 
+.playerStatContainer {
+  position: absolute;
+}
+
+.playerStatName {
+  position: absolute;
+  color: #fff;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.mouseFunctionIcons {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 20px;
+  height: 60px;
+  background-image: url(mouseFunctionIcons.png);
+}
+
+.winningFunction {
+  box-shadow: inset 0 0 2px white;
+  border: 1px solid white;
+  box-sizing: border-box;
+}
+
+.vizTabContent {
+  position: relative;
+  border: 1px solid #ddd;
+  width: 400px;
+  min-height: 400px;
+}
+
+#charts {
+  overflow-x: hidden;
+  overflow-y: auto;
+  height: 400px;
+  max-height: 808px;
+  resize: vertical;
+}
+
 #tabarea {
   position: fixed !important;
 }
 
+@keyframes bounceIn{
+  0%{
+    opacity: 0;
+    transform: scale(0.3);
+  }
+  50%{
+    opacity: 0.9;
+    transform: scale(1.1);
+  }
+  80%{
+    opacity: 1;
+    transform: scale(0.89);
+  }
+  100%{
+    opacity: 1;
+    transform: scale(1);
+  }
+}
+
+@keyframes fadeOut {
+  from {opacity:1;}
+  to {opacity:0;}
+}
+
+.mouse {
+  position: absolute;
+}
+
+.deadMouse {
+
+}
+
+.tooltip {
+  font-size: 12px;
+}
+
+
+@keyframes antiClockwiseSpin {
+  0%  {
+    transform: rotate(360deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+}
+
+@keyframes clockwiseSpin {
+  0%  {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.dust {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  opacity:1;
+  animation: fadeIn 1500ms ease-out;
+  animation-iteration-count: 1;
+}
+
+.cloud1 {
+  animation: clockwiseSpin 4s infinite linear;
+}
+
+.cloud2 {
+  animation: antiClockwiseSpin 4s infinite linear;
+}
+
+.cloud {
+  position: absolute;
+  width: 50px;
+  height: 30px;
+  top: 10px;
+  background: #dde7ee;
+  border-radius: 50%;
+  opacity: .6;
+}
+
+.cloud:before, .cloud:after {
+  content: '';
```

Comment:
yes, the content line is needed. However I am considering replacing the css heart with an image instead which would shorten this css file.